tools/rosbash/scripts/rosrun

```diff
@@ -16,13 +16,13 @@ function debug() {
 
 while [ $args == 1 ]
 do
-  case $1 in
+  case "$1" in
```

Comment:
The first argument is the package name which can by definition not contain spaces.